src/axes/numericAxis.ts

```diff
@@ -262,6 +278,26 @@ export module Axes {
       }
     }
 
+    /**
+     * Gets the approximate text width setting.
+     *
+     * @returns {boolean} The current approximate text width setting.
+     */
+    public approximateTextWidth(): boolean;
+    /**
+     * Sets the approximate text width setting.
+     *
+     * @param {boolean} enable faster but less accurate computation of text widths.
+     */
+    public approximateTextWidth(enable: boolean);
+    public approximateTextWidth(enable?: boolean): boolean {
+      if (enable == null) {
+        return this._approximateTextWidth;
+      } else {
```

Comment:
This is not that great of a deal, but I personally am not a fan of the `else` statement here mainly because it forces a nesting for very little reason.